docs/project-scope.md

```diff
@@ -0,0 +1,161 @@
+# Voyager T800 – Project Scope
+
+A generative AI-powered multimodal travel planning assistant developed during a 12-week Data Science internship.
+
+---
+
+## 1. Project Vision
+
+**Goal**:  
+To build an AI assistant that generates personalized, fact-grounded travel itineraries using both **text** and **image** inputs, powered by LLMs and enhanced with retrieval-augmented generation (RAG), agentic reasoning, and vision-based tools.
+
+---
+
+## 2. Target Users
+
+The primary users of the application:
+
+- [ ] Families planning vacations
+- [ ] Budget-conscious solo travelers
+- [ ] Photographers / travel bloggers
+- [ ] First-time city explorers
+- [ ] Adventure seekers / active travelers
+- [ ] International travelers
+- [ ] Couples planning romantic getaways
+- [ ] Locals exploring nearby destinations
+- [ ] Students on cultural trips
+- [ ] Other: `__________`
```

Comment:
Consider grouping user types into broader categories (e.g., Leisure, Adventure, Local Explorers) for easier reference in product discussions.